website/docs/providers/terraform/d/remote_state.html.md

```diff
@@ -46,6 +46,30 @@ resource "aws_instance" "foo" {
 }
 ```
 
+## Example Usage for local backend
```

Comment:
```suggestion
## Example Usage (`local` Backend)
```

Explanations for these edits: 

- Using code span to indicate we mean the specific backend whose name is "remote" or "local", rather than a category of backends (which is another common meaning of the phrase "remote backend"). 
- With the exception of words that are in code spans, headers should be in title case.